### ISSUE TITLE
keyboard input problem

### ISSUE THREAD

**Issue by SSV-Normandy** [RPCS3/rpcs3#2837]
Project DIVA F[BLJM60527] 
Windows 10 1703 x64 Vulkan 1280x720
i3 4150 and GTX960 with drive 382.22 and GeForce Experience
when using the keyboard input the ‘hold’ input will be interrupted in Diva F especially the hold is very long and keyboard input seems to have more latency than the old version

**Comment by zminhquanz** [comment]
Me too , i think it's a QT GUI issues , i map the numpad key but it mapped about the number key

**Comment by toccata10** [comment]
Archlinux here. On StreetFighter2HD, when I keep the down arrow pressed to crouch, the command won't stick (or hold). My character starts going up, then down, and so on....: it always wants to stand, 
even if I don't release the key. I don't have this non-holding problem on other games (outside rpcs3).

**Comment by Megamouse** [comment]
we might need to use either windows virtual keys for that or native scan code (basically the same?).

The stand up issue could be fixed by filtering autorepeat keys in GSFrame keyPressEvent() with 

if (event->key().isAutoRepeat()) event->ignore();

**Comment by Megamouse** [comment]
Can you please retry now @toccata10 @SSV-Normandy

**Comment by toccata10** [comment]
on version 5350 (from this evening):
1) the stand up issue seems solved (I can't reproduce it anymore), so when I keep a key pressed, it's not interrupted anymore.
2) from time to time, the input "sticks": if I press down, then release, the character stays crouched. I have to make another input to "unstick" the previous one. It seems to appear randomly. It can be with any direction. I tested with street fighter 2 hd and 1942: same problem, from time to time.
3) I can't really tell about latency. It doesn't feel super snappy, but I can't really tell. Maybe when the joystick input for linux will be ready, I'll be able to compare.
So, for me, it's improving with occasional problem.
I tested on master as I didn't see any PR adressing this pb.
